### ISSUE TITLE
Frequent delay in starting searches

### ISSUE THREAD

**Issue by akaHeimdall** [topfunky/PeepOpen-Issues#67]
I often find myself running the Command+T (Textmate) and typing a search string, only to have the first 5 or 6 letters show up in my document before PeepOpen starts up. Although it may not seem like a big issue when one is used to moving so quickly with keyboard commands this results in many rogue keystrokes popping up in my code, then I have to check the open document, and go back to the search, thus impacting productivity. It's just a little nuance that potentially adds up when you don't realize it's happening.

So, in summary about once every five or so times I notice that it takes PeepOpen a few seconds to spin up.

**Comment by takeo** [comment]
i was noticing this a lot as well... I recently got an SSD and it hasn't happened since though. I know that's an expensive fix, but just FYI.

**Comment by topfunky** [comment]
The upcoming v0.2.1 searches in a background thread and displays a progress bar while searching. Other than the initial boot, it should be much faster to launch.

**Comment by chmurph2** [comment]
I have v0.2.1 installed and it still (often) has a delay before it accepts my keystrokes, leading to the problem imarichardson describes.  I created another issue since this one was closed: http://github.com/topfunky/PeepOpen-Issues/issues/issue/94
